<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdressWindow</class>
 <widget class="QDialog" name="AdressWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_Country">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Country:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Town">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Town:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Street">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Street:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Number">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Country">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>28</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Town">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Street">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Number">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>190</y>
     <width>98</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancale">
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>190</y>
     <width>103</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
